napalm_ansible/modules/napalm_install_config.py

```diff
@@ -177,7 +177,10 @@
 def save_to_file(content, filename):
     f = open(filename, 'w')
     try:
-        f.write(content)
+        if type(content) is bytes:
```

Comment:
I think `if isinstance(content, bytes)` would be better than comparing types.

Reply:
agreed!